packages/geo/src/lib/directions/directions.component.ts

```diff
@@ -111,13 +118,33 @@ export class DirectionsComponent implements OnInit, OnDestroy {
 
   constructor(
     private cdRef: ChangeDetectorRef,
+    private http: HttpClient,
     private languageService: LanguageService,
     private directionsService: DirectionsService,
+    private directionsSourceService: DirectionsSourceService,
     private searchService: SearchService,
-    private queryService: QueryService
+    private queryService: QueryService,
+    private authService: AuthService,
+    private messageService: MessageService
   ) {}
 
   ngOnInit(): void {
+    this.authenticated$$ = this.authService.authenticate$.subscribe((auth: boolean) => {
+      if (auth) {
+        const userVerifUrl: string = this.directionsSourceService.sources.find(source => source.type === 'private')?.userVerifUrl;
+        this.http.get(userVerifUrl).subscribe((user: any) => {
+          this.toggleTypeAvailable = user.hasOsrmPrivateAccess;
```

Comment:
J'aurais tendances a nommer la propriété de la ma manière que pour celle du user soit, ça nous donnerait une meilleur lisibilité.
```suggestion
          this.hasOsrmPrivateAccess = user.hasOsrmPrivateAccess;
```